scripts/modelbench/inductor_test.sh

```diff
@@ -15,18 +15,28 @@ LOG_DIR=${5:-inductor_log}
 WRAPPER=${6:-default}
 HF_TOKEN=${7:-hf_xx}
 BACKEND=${8:-inductor}
+# Test Mode
+TEST_MODE=${9:-inference}
+SUITE=${10:-all}
 # Easy to enbale new test
-EXTRA=${9}
+EXTRA=${11}
 
 mkdir -p $LOG_DIR
 
 export HUGGING_FACE_HUB_TOKEN=${HF_TOKEN}
 # https://github.com/pytorch/pytorch/issues/107200
 pip uninstall transformers -y && pip install transformers==4.30.2
+# fix issue: AttributeError: module 'importlib.resources' has no attribute 'files'
+pip uninstall networkx -y && pip install networkx
 
 # skip sam & nanogpt_generate for stable results
 sed -i '/skip_str = " ".join(skip_tests)/a\    skip_str += " -x sam -x nanogpt_generate"' benchmarks/dynamo/runner.py
 
+if [[ ${TEST_MODE} == "training_full" ]]
+    # skip hf_GPT2_large, cuz it will OOM after using jemalloc
+    sed -i "/SKIP_TRAIN = {/a\    \"hf_GPT2_large\"," benchmarks/dynamo/torchbench.py
```

Comment:
whether this list can effect when the test launched by runner.py? And is there no more OOM models in other 2 suites?

Reply:
Yes, this model is the only 1 model which will OOM. And I verified this several times before. Cuz runner will read this list `SKIP_TRAIN` before executing training.